controller/scheduler/formation.go

```diff
@@ -44,6 +54,15 @@ func (p Processes) IsEmpty() bool {
 	return true
 }
 
+func (p Processes) IsScaleDown() bool {
+	for _, count := range p {
+		if count < 0 {
```

Comment:
Should this check if the process is being scaled all the way to zero? There are reasons to scale down say the controller from 2 to 1 without a new release.